cmd/state-tray/main.go

```diff
@@ -16,32 +17,56 @@ func main() {
 }
 
 func onReady() {
+	err := run()
+	if err != nil {
+		fmt.Fprintln(os.Stderr, err.Error())
+		logging.Error("Systray encountered an error: %v", err)
+		os.Exit(1)
+	}
+}
+
+func run() error {
 	box := packr.NewBox("assets")
 	systray.SetIcon(box.Bytes("icon.ico"))
-	systray.SetTooltip("ActiveState State Tool")
+	systray.SetTooltip(locale.Tl("tray_tooltip", "ActiveState State Tool"))
 
-	mAbout := systray.AddMenuItem("About State Tool", "Information about the State Tool")
+	mAbout := systray.AddMenuItem(
+		locale.Tl("tray_about_title", "About State Tool"),
+		locale.Tl("tray_about_tooltip", "Information about the State Tool"),
+	)
 
 	systray.AddSeparator()
 
-	mDoc := systray.AddMenuItem("Documentation", "Open State Tool Docs")
+	mDoc := systray.AddMenuItem(
+		locale.Tl("tray_documentation_title", "Documentation"),
+		locale.Tl("tray_documentation_tooltip", "Open State Tool Docs"),
+	)
 
-	mPlatform := systray.AddMenuItem("ActiveState Platform", "")
-	mLearn := mPlatform.AddSubMenuItem("Learn", "Open ActiveState blog")
-	mSupport := mPlatform.AddSubMenuItem("Support", "Open support page")
-	mAccount := mPlatform.AddSubMenuItem("Account", "Open your account page")
+	mPlatform := systray.AddMenuItem(locale.Tl("tray_platform_title", "ActiveState Platform"), "")
+	mLearn := mPlatform.AddSubMenuItem(
+		locale.Tl("tray_learn_title", "Learn"),
+		locale.Tl("tray_learn_tooltip", "Open ActiveState blog"),
+	)
+	mSupport := mPlatform.AddSubMenuItem(
+		locale.Tl("tray_support_title", "Support"),
+		locale.Tl("tray_support_tooltip", "Open support page"),
+	)
+	mAccount := mPlatform.AddSubMenuItem(
+		locale.Tl("tray_account_title", "Account"),
+		locale.Tl("tray_account_tooltip", "Open your account page"),
+	)
 
 	systray.AddSeparator()
 
-	mQuit := systray.AddMenuItem("Exit", "")
+	mQuit := systray.AddMenuItem(locale.Tl("tray_exit", "Exit"), "")
 
 	for {
 		select {
 		case <-mAbout.ClickedCh:
 			logging.Debug("About event")
 			err := open.Prompt("state --version")
 			if err != nil {
-				handleError(errs.Wrap(err, "Could not open command prompt"))
+				return errs.Wrap(err, "Could not open command prompt")
```

Comment:
Looking at this again now, this feels like it should just be logging the error. Since we don't want to end things just cause the "About" click caused an error.

Reply:
Makes sense. What do you think about setting logging to verbose by default here so that any calls to `logging.Error` also print? That way we don't have to do the additional call to `fmt.Fprintln`

Reply:
Sure, I think that makes sense.